### PR TITLE
Header component mobile menu & snapshot

### PR DESCRIPTION
**Pull request by 0750kosse**

### CODE REVIEW

**Comment by 0750kosse** [comment]
LGTY...?

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
```
Comment (t0nin0s) [inline src/components/Header/Header.tsx:6]: app not running error below:
./src/components/Header/Header.tsx
Module not found: Can't resolve '@material-ui/icons/Menu'

maybe you installed the package globally?

**Review by t0nin0s — CHANGES_REQUESTED** [review]
cant run the app, error below:
./src/components/Header/Header.tsx
Module not found: Can't resolve '@material-ui/icons/Menu'

maybe you installed the package globally?

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (0750kosse) [reply src/components/Header/Header.tsx:6]: Review pls..?

**Review by t0nin0s — APPROVED** [review]
LGTM for devices, we would need to refactor this for web as well so menu items are displayed along with the page title

**Pull request merged**